tests/passes/partitioning/test_parts.py

```diff
@@ -30,7 +30,8 @@ def big_qasm_file(request: Any) -> str:
 
 
 def test_parters(big_qasm_file: str, compiler: Compiler) -> None:
-    if 'barrier' not in big_qasm_file:
+    keywords = ['barrier', 'reset', 'measure']
+    if all(keyword not in big_qasm_file for keyword in keywords):
         return  # TODO: REMOVE THIS WHEN BARRIER IS FIXED
```

Comment:
My bad, I forgot to remove this. You should remove this if check altogether, so that we can properly test these circuits.